src/web3py/extensions/contracts.py

```diff
@@ -154,10 +173,20 @@ def get_accounting_last_processing_ref_slot(self, blockstamp: BlockStamp) -> Slo
         FRAME_PREV_REPORT_REF_SLOT.labels('accounting').set(result)
         return result
 
+    # TODO: Why here's no cache applied?
```

Comment:
it was done explicitly 🤔  https://github.com/lidofinance/lido-oracle/pull/289#discussion_r1148337913

Reply:
might be because it is used only in one particular place rather than the accounting thing

Reply:
🤔

Reply:
Yeah, it's called once. Only to collect metrics.